### PR TITLE
[#419] use onError instead of react-imageloader

### PR DESCRIPTION
**Pull request by k88hudson**
According to the docs, `onError` and `onLoad should be supported on image elements in React, but they don't appear to be working, so I had to attach the event manually.

@gvn what do you think?

### CODE REVIEW

File: src/components/card/card.jsx [hunk 1]
```diff
@@ -11,16 +10,20 @@ var Card = React.createClass({
...
-  prerenderImage: function() {
-    return React.createElement('img', {
-      src: Card.DEFAULT_THUMBNAIL
-    });
+  onImageError: function() {
+    var imageEl = this.refs.imageEl.getDOMNode();
+    imageEl.src = Card.DEFAULT_THUMBNAIL;
+    imageEl.onerror = null;
+  },
+  componentDidMount: function () {
```
Comment (gvn) [inline src/components/card/card.jsx:18]: If possible we should bind before setting the `src` to prevent race conditions.

Comment (k88hudson) [inline src/components/card/card.jsx]: i guess we could create the image element here too. do you think that's better? i can't think of another way to do it

**Comment by gvn** [comment]
I think manually attaching is fine. This approach also removes a dep, which is nice.

File: src/components/card/card.jsx (hunk 1 above)
Comment (k88hudson) [inline src/components/card/card.jsx]: hm, or adding the src here too

File: src/components/card/card.jsx [hunk 2]
```diff
@@ -11,16 +10,21 @@ var Card = React.createClass({
...
+    var imageEl = this.refs.imageEl.getDOMNode();
+    imageEl.onerror = this.onImageError;
+    imageEl.src = this.props.thumbnail || Card.DEFAULT_THUMBNAIL;
```
Comment (k88hudson) [inline src/components/card/card.jsx:21]: ok what do you think about this?

Comment (gvn) [inline src/components/card/card.jsx:21]: :+1: LGTM!

**Pull request merged**